tests/test_feature_aggregate.py

```diff
@@ -299,6 +312,26 @@ def test_aggregate_region_with_components(simple_df):
     assert _df.check_aggregate_region(v, components=["foo"]) is None
 
 
+@pytest.mark.parametrize(
+    "data, variable, weight",
+    ((NEG_WEIGHTS_DF, "Price|Carbon", "Emissions|CO2"),),
+)
+def test_agg_weight(data, variable, weight):
+
```

Comment:
Using pytest-parametrization here is a bit overkill (unless you plan to add more versions of this test?)
```suggestion
def test_agg_weight():
    data = NEG_WEIGHTS_DF
    variable = "Price|Carbon"
    weight = "Emissions|CO2"
```